codegen/masm/src/config.rs

```diff
@@ -51,6 +51,9 @@ pub struct CodegenConfig {
     /// Memory range used to store exponentiations of Z, each address contains one point to be used
     /// on the evaluation of each periodic polynimal.
     pub z_exp_address: u32,
+
+    /// Memory position of the domain generator
+    pub domain_generator_address: u32,
```

Comment:
This is the LDE domain generator, right? I may be good to differentiate between LDE and trace domain generator (the relationship is $g_{lde}^3 = g_{trace}$, I believe).

Reply:
It is $g_{lde}^{2^3} == g_{trace}$, basically  $g_{lde}^{blowup} == g_{trace}$

Reply:
ah - yes! Sorry for the confusion!

Reply:
> This is the LDE domain generator, right?

This is the trace domain generator. I will add a `trace_` prefix.